src/hssm/datasets.py

```diff
@@ -19,7 +19,12 @@
         filename="cavanagh_theta",
         path=os.path.join(base_dir, "hssm/datasets/cavanagh_theta_nn.csv"),
         description="Description for cavanagh_theta dataset",
-    )
+    ),
+    "cavanagh_theta_old": FileMetadata(
```

Comment:
why do we actually need the old version?

Reply:
For backward compatibility. Some people might have already written some code with the old dataset. This way they can adapt their code to the new version without much trouble. We can remove this in the future.